### PR TITLE
Remove unused PoA<>Substrate bridge

### LINKED ISSUE #1208: Remove unused ETH PoA bridge
There are currently no plans to productize the Ethereum PoA bridge.

We should remove the code as it brings maintenance overhead. We can bring it back at a later time if/when there is bandwidth or need for it.

### PR DESCRIPTION
**Pull request by acatangiu**
There are currently no plans to productize the Ethereum PoA <> Subtrate bridge.

This PR remove the code as it brings maintenance overhead.

We can bring the code back at a later time if/when there is bandwidth or need for it.

Fixes #1208, makes #117 and #416 obsolete.

### CODE REVIEW

**Comment by acatangiu** [comment]
I guess we should also tag the repo before merging this PR, so we have a _last available `B-PoA<>Substrate` version_.

File: deployments/run.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 # To deploy a network you can run this script with the name of the bridge (or multiple bridges) you want to run.
 #
-# `./run.sh poa-rialto rialto-millau`
+# `./run.sh westend-rialto rialto-millau`
```
Comment (svyatonik) [inline deployments/run.sh:7]:
```suggestion
# `./run.sh westend-millau rialto-millau`
```

File: deployments/README.md [hunk 2]
```diff
@@ -44,16 +44,16 @@ the monitoring Compose file is _not_ optional, and must be included for bridge d
...
-1. Ethereum PoA to Rialto Substrate
-2. Rialto Substrate to Millau Substrate
+1. Rialto Substrate to Millau Substrate
+2. Westend Substrate to Rialto Substrate
 
 These bridges can be deployed using our [`./run.sh`](./run.sh) script.
 
 The first argument it takes is the name of the bridge you want to run. Right now we only support two
-bridges: `poa-rialto` and `rialto-millau`.
+bridges: `rialto-millau` and `westend-rialto`.
```
Comment (svyatonik) [inline deployments/README.md:53]:
```suggestion
bridges: `rialto-millau` and `westend-millau`.
```

File: deployments/README.md [hunk 3]
```diff
@@ -44,16 +44,16 @@ the monitoring Compose file is _not_ optional, and must be included for bridge d
 
 ### Running and Updating Deployments
 We currently support two bridge deployments
-1. Ethereum PoA to Rialto Substrate
-2. Rialto Substrate to Millau Substrate
+1. Rialto Substrate to Millau Substrate
+2. Westend Substrate to Rialto Substrate
```
Comment (svyatonik) [inline deployments/README.md:48]:
```suggestion
2. Westend Substrate to Millau Substrate
```

**Review by svyatonik — APPROVED** [review]
Please also do/coordinate a couple of things:
1) archive/remove https://github.com/paritytech/openethereum repository;
2) remove `relays/headers` folder (probably in a separate PR) - now it is only used to expose `SyncLoopMetrics`. We may move this struct to the finality relay && only keep `best_block_numbers` (as the `blocks_in_state` is not relevant anymore).

**Pull request merged**